WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -51,55 +54,55 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
-    /** Instantiate the interface and set the context  */
-    class WebAppInterface(private val mContext: Context) {
-        /** Send a message from the web page  */
-        @JavascriptInterface
-        fun sendMessage(message: String) {
-            val sendIntent: Intent = Intent().apply {
-                action = Intent.ACTION_SEND
-                putExtra(Intent.EXTRA_TEXT, message)
-                type = "text/plain"
-            }
-            val shareIntent = Intent.createChooser(sendIntent, null)
-            startActivity(mContext, shareIntent, null)
+    private fun createJsObject(
+        webview: WebView,
+        mContext: Context,
+        jsObjName: String,
+        rules: Set<String>,
```

Comment:
nit: `allowedOriginRules` would be a clearer name.

Reply:
Ok